### PR TITLE
master-lps-107366

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-107366

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

File: modules/apps/redirect/redirect-test/src/testFunctional/tests/Redirect.testcase [hunk 1]
```diff
@@ -942,6 +942,35 @@ definition {
 			locator1 = "Home#PAGE_ACTIVE");
 	}
 
+	@description = "This ensures that the old friendlyURL can be removed from the history."
+	@priority = "5"
+	test RemoveOldFriendlyURL {
+		property osgi.module.configuration.file.names = "com.liferay.layout.admin.web.internal.configuration.FFViewFriendlyURLHistoryConfiguration.config";
```
Comment (austinchiang) [inline modules/apps/redirect/redirect-test/src/testFunctional/tests/Redirect.testcase:948]: @Jeremy-Ch once our devs enable the feature flag, we won't need this property anymore. Can you create a test improvement ticket to track that for later?

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/liferay-lima/liferay-portal/pull/84<br></br>

**Pull request closed without merging**